cdn/dev/css/template.css

```diff
@@ -2176,7 +2176,7 @@ button.feedback{
 
 @media all and (min-width: 10px) and (max-width: 600px){
 	.wrapper{
-		width: 320px;
+		width: auto;
```

Comment:
Removes that left and right margin on small devices which is both ugly and unhelpful.